types/handsontable/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Handsontable 0.31
+// Type definitions for Handsontable 0.31.2
```

Comment:
The types declared for `afterChange` are specific to 0.31.2. The types for 0.31.1 appeared to by `[number, string, string, any][]` (row, prop, oldVal, newVal), as it previously used the `prop` value rather than the `col` value. I'd be happy to change the signature to `[number, any, any, any][]` to make this slightly more adaptable to different versions.